xbmc/commons/ilog.h

```diff
@@ -53,6 +53,7 @@
 #define LOGCEC        (1 << (LOGMASKBIT + 9))
 #define LOGVIDEO      (1 << (LOGMASKBIT + 10))
 #define LOGWEBSERVER  (1 << (LOGMASKBIT + 11))
+#define LOGTIMING     (1 << (LOGMASKBIT + 12))
```

Comment:
not sure if such a generic specifier is a good idea. it will get used by different components and you'll end up seeing a log of python log spam too.